File: two-sum/two_sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        object = {}
        for idx in range(0, len(nums)):
            if (target-nums[idx]) in object:
                return [object[target-nums[idx]], idx]
            object[nums[idx]] = idx

File: two-sum/two_sum.js
const twoSum = (nums, target) => {
    object = {}
    for (let i=0; i <= nums.length; i++) {
        if (object.hasOwnProperty(target-nums[i])) {
            return [object[target-nums[i]], i];
        }
        object[nums[i]] = i
    }
};
